src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double longerBase;
+    private final double shorterBase;
+    private final double height;
+
+    public IsoscelesTrapezoid(double longerBase, double shorterBase, double height, String color) {
+        super(color);
+        this.longerBase = longerBase;
+        this.shorterBase = shorterBase;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((shorterBase + longerBase) * height) / 2;
+    }
+
+    @Override
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. In this case, consider separating the drawing behavior into a separate interface.